src/sly_unpacker.cpp

```diff
@@ -1,53 +1,55 @@
 #include "types.hpp"
 #include "fs.hpp"
-#include "stream_utils.hpp"
 #include "wac.hpp"
 
 #include <algorithm>
 #include <fstream>
 #include <iterator>
 #include <stdexcept>
+#include <filesystem>
 
 int main(int argc, char *argv[]) {
     try {
-        if (argc <= 2)
+        if (argc < 2)
             throw std::runtime_error(std::string(argv[0]) + " <input_file> [<output_dir>]");
 
-        const std::string wac_path = argv[1];
-        const std::string output_path = (argc == 3) ? argv[2] : wac_path + "/extracted";
+        const std::string wac_path_string = argv[1];
+        const std::filesystem::path wac_path{ wac_path_string };
```

Comment:
```suggestion
        const std::filesystem::path wac_path(argv[1]);
```
and remove the above line